trade-finance-manager-api/src/v1/api.js

```diff
@@ -820,13 +821,35 @@ const getPartyDbInfo = async ({ companyRegNo }) => {
       url: `${EXTERNAL_API_URL}/party-db/${encodeURIComponent(companyRegNo)}`,
       headers: headers.external,
     });
-    return response.data;
+    return AUTOMATIC_SF_CUSTOMER_CREATION_ENABLED ? { status: 200, data: response.data } : response.data;
```

Comment:
the response needs a status (following the pattern elsewhere in dtfs) so that we can distinguish between a 404 and an actual failure when querying SF - we don't want to create customers unless we definitely have a 404